packages/liferay-npm-scripts/src/jest/mocks/liferay.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const authToken = 'default-mocked-auth-token';
+
+/**
+ * Event support APIs on the `Liferay` object inherited from `A.Attributes`
+ *
+ * Defines:
+ * 	- Liferay.detach
+ * 	- Liferay.fire
+ * 	- Liferay.on
```

Comment:
I don't think you need this list because the defined things are visible immediately below (and the list is wrong anyway because it doesn't include `Liferay.once`).

Reply:
it’s because I added it afterwards... it proves why documentation is usually wrong 😂😂

Reply:
**Annoying TypeScript Advocate:** Documentation is mostly redundant in the presence of a robust type system.

![source (1)](https://user-images.githubusercontent.com/7074/63749429-b6011d80-c8ab-11e9-9ae9-12fddf23c283.gif)